### PR TITLE
Update UI component in DG

### PR DESCRIPTION
**Pull request by shogun187**
Separated the main UI class diagram and the OutputPanel class diagram for better readability

### CODE REVIEW

File: docs/DeveloperGuide.md [hunk 1]
```diff
@@ -73,7 +73,15 @@ The **API** of this component is specified in [`Ui.java`](https://github.com/se-
...
 ![Structure of the UI Component](images/UiClassDiagram.png)
 
-The UI consists of a `MainWindow` that is made up of parts e.g.`CommandBox`, `ResultDisplay`, `PersonListPanel`, `StatusBarFooter` etc. All these, including the `MainWindow`, inherit from the abstract `UiPart` class which captures the commonalities between classes that represent parts of the visible GUI.
+
+
+
+The UI consists of a `MainWindow` that is made up of parts e.g. `OutputPanel`,`CommandBox`, `ResultDisplay`, `PersonListPanel`, `StatusBarFooter` etc. All these, including the `MainWindow`, inherit from the abstract `UiPart` class which captures the commonalities between classes that represent parts of the visible GUI.
+
+Additionally, the structure of `OutputPanel` is as shown below:
+![Structure of the OutputPanel Component](images/OutputPanelClassDiagram.PNG)
```
Comment (izzahaj) [inline docs/DeveloperGuide.md:82]:
```suggestion
![Structure of the OutputPanel Component](images/OutputPanelClassDiagram.png)
```

**Review by izzahaj — CHANGES_REQUESTED** [review]
One small capitalisation issue in the extension name of one of your diagrams.

File: docs/DeveloperGuide.md (hunk 1 above)
Comment (shogun187) [reply docs/DeveloperGuide.md]: Updated

**Review by izzahaj — APPROVED** [review]
LGTM

**Pull request merged**